Modelica/Clocked/Examples.mo

```diff
@@ -0,0 +1,5193 @@
+(here: \"T_c\"). This is performed with the help of block
+<a href=\"modelica://Modelica.Blocks.Math.InverseBlockConstraints\">Modelica.Blocks.Math.InverseBlockConstraints</a>
+that allows connecting an external input to an output
+in the pre-filter design block
+<a href=\"modelica://Modelica.Clocked.Examples.Systems.Utilities.ComponentsMixingUnit.FilterOrder\">Utilities.ComponentsMixingUnit.FilterOrder</a>:
+</p>
+
+<p>
+<img src=\"modelica://Modelica/Resources/Images/Clocked/Examples/ControlledMixingUnit_FilterDesign.png\">
+</p>
+
+
+<p>
+Translating this
+model will generate the continuous-time inverse plant model.
+However, a Modelica tool will give an error message that it
+has to differentiate the model, but this requires the
+<b>second derivative</b> of the external input c_ref and this derivative
+is not available. The conclusion is that a low pass filter of at
+least second order has to be connected between c_ref and c, for example
+<a href=\"modelica:Modelica.Blocks.Continuous.Filter\">Modelica.Blocks.Continuous.Filter</a>. Only filter types should be used that do not have \"vibrations\" in the time domain for
+a step input. Therefore, parameter <b>analogFilter</b> of the component should be
+selected as <b>CriticalDamping</b> (= only real poles), or
+<b>Bessel</b> (= nearly no vibrations, but steeper frequency response as
+CriticalDamping). The cut-off frequency <b>f_cut</b> is manually
+selected by simulations of the closed loop system. In the example,
+a CriticalDamping filter of third order (the third order is selected to
+get smoother signals) and a cut-off frequency of 1/300 Hz is used.
+</p>
+
+<h4>Design of Controller</h4>
+<p>
+The controller for the mixing unit is shown in the diagram layer of block
+at hand, as
+well as in the following figure:
+</p>
+
+<p>
+<img src=\"modelica://Modelica/Resources/Images/Clocked/Examples/ControlledMixingUnit_Controller.png\">
+</p>
+
+It consists of the filter discussed above. The input to the filter is the reference
+concentration which is filtered by the low pass filter. The output of the filter
+is used as input to the concentration c in the inverse plant model.
+This model computes the desired cooling temperature T_c (which is used
+as desired cooling temperature at the output of the controller)
+and the desired temperature T (which is used as desired value for
+the feedback controller). This part of the control system is the
+\"feed-forward\" part that computes the desired actuator signal.
+As feedback controller a simple P-Controller with one gain is used.
+</p>
+
+<p>
+This controller could be defined as continuous-time system in previous Modelica
+versions. However, with Modelica 3.3 it is now also possible to define the
+controller as sampled data system. For this, the two inputs are sampled
+(sample1 and sample2) and the actuator output is hold (hold1).
+The controller partition is then associated with a periodic clock
+(via sample2) that has a sample 
```

Comment:
Do not use the obsolete icon (here and in other blocks), but use Modelica.Blocks.Icons.Block instead.